developing-without-a-build/article.md

```diff
@@ -0,0 +1,266 @@
+# Developing Without a Build
+In the early days of web development, all we needed was a simple file editor and a web server. It was easy for a newcomer to understand the process and get started making their own web pages. Web development has changed a lot since then: the complexity of the tools we use for development has grown just as much as the complexity of the things that we're building on the web.
+
+Nowadays, developers can set up systems which transform their code from one syntax to another, turning it into code that actually runs in the browsers. To aid debugging, source maps need to be configured carefully between the different tools, so that we can still recognize and debug our code in the browser.
+
+Imagine whats it's like for a newcomer or even an experienced developer:
+- They first need to learn a lot of different tools before they can get their code to actually run
+- Their IDE and linter won't understand the syntax of this new framework that was recommended to them by a friend, so they have to look for some extra plugins
+- After that they run into some more errors... Turns out they wanted to keep it simple and not use typescript, but they didn't configure the babel plugins for decorators correctly in the right order so the project didn't compile
+
+It may sound exaggerated, but this experience is common to a lot of developers. You may have jumped through similar hoops yourself.
+
+I feel that's really a shame. One of the key selling points of the web is and always was that's an easy and open format. It should be easy to just get started right away without a lot of configuration, tooling, pomp, and ceremony.
```

Comment:
```suggestion
I feel that's really a shame. One of the key selling points of the web is and always was that it's an easy and open format. It should be easy to just get started right away without a lot of configuration, tooling, pomp, and ceremony.
```

pomp?

Reply:
faff

Reply:
faff?

Reply:
Huh. TIL.

(there's a typo fix in the suggestion btw, dont forget)

Reply:
Thanks. Made the pomp faff bit shorter :)